scripts/seed_datastore.py

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env python
+#
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Fetches features from the staging or live dashboard site, and inserts them into the development
+datastore.
+
+This script can also set a user as admin, to facilitate testing those parts of the site.
+"""
+
+from datetime import datetime
+from google.cloud import ndb
+from urllib.parse import urljoin
+import argparse
+import logging
+import os
+import pprint
+import requests
+import sys
+
+sys.path = [os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
+            ] + sys.path
+os.environ['DATASTORE_EMULATOR_HOST'] = 'localhost:15606'
+os.environ['GAE_ENV'] = 'localdev'
+os.environ['GOOGLE_CLOUD_PROJECT'] = 'cr-status-staging'
+os.environ['SERVER_SOFTWARE'] = 'gunicorn'
+
+from internals.core_enums import MISC
+from internals.core_models import FeatureEntry
+from internals.user_models import AppUser
+
+parser = argparse.ArgumentParser(description='seed the development datastore')
+parser.add_argument(
+    '--server',
+    help='the root of a chrome status server, or "" to skip importing',
+    default='https://cr-status-staging.appspot.com/')
+parser.add_argument('--admin',
+                    help='email address of the user to make a site admin')
+parser.add_argument(
+    '--after',
+    type=datetime.fromisoformat,
+    help='only add features that have been updated since AFTER',
+    default='0001-01-01')
+
+args = parser.parse_args()
+client = ndb.Client()
+with client.context():
+    if args.server:
+        target = urljoin(args.server, 'features.json')
```

Comment:
Could you move this section into a separate function e.g. `def add_features` ?

Reply:
@DanielRyanSmith @KyleJu @jrobbins Is the features.json API also going away soon? Or only the old features page? I was just wondering because of the google cloud alerts. If so, I'd rather use an alternative API if the features.json API is slated to go away soon.

Reply:
Done.

Reply:
> Is the features.json API also going away soon?

After talking with the team, it will be supported for awhile